instana/json_span.py

```diff
@@ -122,7 +122,7 @@ def __init__(self, **kwds):
 
 class SDKData(object):
     name = None
-    Type = None
+    type = None
     arguments = None
     Return = None
```

Comment:
Nice catch. Shouldn’t we do the same with `Return`? Also: I think we should fix it backend side too as this does not handle already compiled GoLang executables.

Reply:
Good point I was so focused on the Type I missed Return. Just committed that too.
I'm just started to look at Golang to see if it has the same problems.
It would be quite a big change on the backend to make it case insensitive and could have performance implications due to the high volume of traces processed.